spec/requests/drafts_controller_spec.rb

```diff
@@ -236,5 +236,27 @@
       expect(response.status).to eq(200)
       expect(Draft.get(user, "xxx", 0)).to eq(nil)
     end
+
+    it "can destroy a draft for a passed user" do
+      sign_in(Fabricate(:admin))
+      user = Fabricate(:user)
+      Draft.set(user, "xxx", 0, "hi")
+      delete "/drafts/xxx.json", params: { sequence: 0, username: user.username, api: true }
+      expect(response.status).to eq(200)
+      expect(Draft.get(user, "xxx", 0)).to eq(nil)
+    end
+
+    it "raises missing user" do
+      delete "/drafts/xxx.json", params: { sequence: 0 }
+      expect(response.status).to eq(500)
```

Comment:
500? That imo should be reserved for unexpected errors that aren't handled. This is something from a 4xx range, like 401 unauthorized